gtsfm/scene_optimizer.py

```diff
@@ -415,6 +417,17 @@ def save_full_frontend_metrics(
                 "inlier_ratio_gt_model": round(report.inlier_ratio_gt_model, PRINT_NUM_SIG_FIGS)
                 if report.inlier_ratio_gt_model
                 else None,
+                "inlier_avg_reproj_error_gt_model": round(
+                    np.nanmean(report.reproj_error_gt_model[report.v_corr_idxs_inlier_mask_gt]), PRINT_NUM_SIG_FIGS
+                )
+                if report.reproj_error_gt_model is not None and report.v_corr_idxs_inlier_mask_gt is not None
+                else None,
+                "outlier_avg_reproj_error_gt_model": round(
+                    np.nanmean(report.reproj_error_gt_model[np.logical_not(report.v_corr_idxs_inlier_mask_gt)]),
```

Comment:
@travisdriver can't we just use `report.inlier_avg_reproj_error_gt_model` here? why do we need to compute the mean over again? I think we already compute it in the TwoViewEstimator

https://github.com/borglab/gtsfm/pull/295/files?authenticity_token=sUeFac35fdQHIM%2FWNeNuKnBbZw%2Bq8phgw%2B4iKHfmflK7LqGME7HzIYxnQRv3OJqxvyVxjM1akbCEnMutH8PLxg%3D%3D&file-filters%5B%5D=.py#diff-1f5b63f35ed4d88709ce154ae9808f38a1a7178fb5f74211b1053ae5a8c37894R177

Reply:
You're right. I think it's cleaner to just compute it here when everything else is computed. I'll remove it here